nomad/fsm.go

```diff
@@ -293,12 +293,11 @@ func (n *nomadFSM) Apply(log *raft.Log) interface{} {
 		return n.applyNamespaceUpsert(buf[1:], log.Index)
 	case structs.NamespaceDeleteRequestType:
 		return n.applyNamespaceDelete(buf[1:], log.Index)
-	case structs.EventSinkUpsertRequestType:
-		return n.applyUpsertEventSink(buf[1:], log.Index)
-	case structs.EventSinkDeleteRequestType:
-		return n.applyDeleteEventSink(buf[1:], log.Index)
-	case structs.BatchEventSinkUpdateProgressType:
-		return n.applyBatchUpdateEventSink(buf[1:], log.Index)
+	// COMPAT(1.0): Allow 1.0-beta clusterers to gracefully handle
```

Comment:
Let's be a bit more descriptive:
```suggestion
	// COMPAT(1.0): These messages were added and removed during the 1.0-beta series and should not be immediately reused for other purposes
```